src/server.js

```diff
@@ -138,20 +144,30 @@ let load = loader({
       // Create server and start listening
       let server = http.createServer(app);
       await new Promise((accept, reject) => {
-        server.listen(cfg.server.port, accept);
+        server.listen(cfg.server.port);
+        server.once('listening', () => {
+          console.log("Listening on port: " + cfg.server.port);
+        });
         server.once('error', reject);
+        server.once('close', accept);
       });
-      console.log("Listening on port: " + cfg.server.port);
+    },
+  },
+
+  scanner: {
+    requires: ['cfg', 'authorizers'],
+    setup: async ({cfg, authorizers}) => {
+      await scanner(cfg, authorizers);
     },
   },
 }, ['profile']);
 
 if (!module.parent) {
-  load('server', {
+  load(process.argv[3], {
     profile: process.argv[2]
```

Comment:
In other project we've started to use `NODE_ENV` as profile...

heroku defaults to `NODE_ENV=production`, it also changes the behavior of a few node modules... notably express, but that isn't super important...

Anyways, stop me if that is a bad idea :) or ideally follow the pattern if you like it...